sphinxcontrib/vcs.py

```diff
@@ -27,10 +27,15 @@ def get_revision(argument):
     return argument.strip()
 
 
+def is_merge(commit):
+    return commit.message[0:6] == 'Merged'
```

Comment:
I'm not sure how to check "Merged commit", but I think the commit message can be changed other message. Is there proper way?